### ISSUE TITLE
Model Architecture related question

### ISSUE THREAD

**Issue by srikar2097** [alexandru-dinu/cae#10]
Hi @alexandru-dinu 
thank you for sharing your work on Lucas et. al. paper. I have the following questions:

1. Did u implement the quantization + Entropy coding (as mentioned in the paper)? I do not see this in the code. In neural network based compression model, these 2 layers are the novelty. If they are absent how is your model able to compress images?
2. How is it possible to arrive at a bpp when training these models (in paper they use 3-bit rates 0.25, 0.372 and 0.5).

thanks.

**Comment by maodouy** [comment]
There is a quantitative training part in the code, but I have not seen the part of the entropy estimate. Therefore, the code cannot control the rate, and only one network model corresponds to a bit rate. For your first question: the autoencoded's encoder outputs the compressed feature, which is  compression. Entropy coding is lossless compression. The second problem: the calculation of the code rate is the output of the encoder except the source image. As far as I know, there are currently two rate control methods: one is the RD curve and the other is the iteration. As far as I know, there are currently two rate control methods: one is the RD curve (the difficulty is the rate estimate, which is the estimate of the entropy. You need to add this part to the loss function of the control), and the other is the iteration (using the RNN). It's easier than using CNN).

**Comment by alexandru-dinu** [comment]
Moving discussion to #15.
